src/System.Net.Http/tests/FunctionalTests/System.Net.Http.Functional.Tests.csproj

```diff
@@ -176,9 +176,6 @@
   <ItemGroup>
     <SupplementalTestData Include="$(PackagesDir)system.net.testdata\1.0.1-prerelease\content\**\*.*" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsOSX)'=='true'">
-    <TestCommandLines Include="ulimit -Sn 4096" />
```

Comment:
Is this dead code? Or why are we removing it?

Reply:
dead code, yes.